bin/prebuild.js

```diff
@@ -0,0 +1,15 @@
+const chalk = require('chalk');
+const git = require('git-describe');
+const path = require('path');
+const fs = require('fs-extra');
+const { version: appVersion } = require('../package.json')
+
+const info = git.gitDescribeSync({ longSemver: true });
+const file = path.resolve(__dirname, '..', 'projects', 'app', 'src', 'version.json');
```

Comment:
Check if we could not just add this file to the root of the folder instead of inside projects/app/src. In the angular.json file you specify the path for this asset, I don't think it needs to be inside projects.

Reply:
> __Warning__
> that is not allowed; 
`An unhandled exception occurred: The projects/app/version.json asset path must start with the project source root.
`
or
`Error: The version.json asset path must start with the project source root.`